helpers/integration.go

```diff
@@ -96,6 +97,11 @@ func (t *TaskRun) GetTestResult() (*HACBSTestResult, error) {
 	if t.testResult != nil {
 		return t.testResult, nil
 	}
+	// load schema for test validation
+	sch, err := jsonschema.Compile("helpers/test_schema.json")
```

Comment:
Hm, I don't think referencing the schema by relative path will work, since functions that call this will be in different places. I'm open to moving the schema back to be a JSON object and we can figure out generating a JSON file for reference in some other way.